ytfzf

```diff
@@ -1217,6 +1275,8 @@ check_if_url "${search_query:=$*}"
 #format the menu screen
 format_menu
 
+[ -n "$sort_name" ] && $sort_name
+unset sort_name
```

Comment:
I guess the check is unecessary because If its unset nothing happens

unset was used everywhere else so I just assumed I could use it but the script should work fine without it

Reply:
my bad thats right

Reply:
well i've removed it so if there's a need to add it back then we can add it back, but if you wanted to check what sort_name was used you have the option when it's not unset